_pages/username.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: mzwong
+pageUrl: blank_url
```

Comment:
@mzwong our tests actually check your URL so they won't allow you to submit a blank url, go ahead and enter your URL from following class, and it should pass!